presto-main/src/main/java/com/facebook/presto/operator/aggregation/BlackholeAggregation.java

```diff
@@ -0,0 +1,137 @@
+
+    public BlackholeAggregation()
+    {
+        super(NAME,
+                ImmutableList.of(comparableTypeParameter("T")),
+                ImmutableList.of(),
+                parseTypeSignature(StandardTypes.VARBINARY),
+                ImmutableList.of(parseTypeSignature("T")));
+    }
+
+    @Override
+    public String getDescription()
+    {
+        return "Consumes given values as a blackhole.";
+    }
+
+    @Override
+    public BuiltInAggregationFunctionImplementation specialize(BoundVariables boundVariables, int arity, FunctionAndTypeManager functionAndTypeManager)
+    {
+        Type valueType = boundVariables.getTypeVariable("T");
+        return generateAggregation(valueType);
+    }
+
+    private static BuiltInAggregationFunctionImplementation generateAggregation(Type type)
+    {
+        DynamicClassLoader classLoader = new DynamicClassLoader(BlackholeAggregation.class.getClassLoader());
+
+        List<Type> inputTypes = ImmutableList.of(type);
+
+        AggregationMetadata metadata = new AggregationMetadata(
+                generateAggregationName(NAME, type.getTypeSignature(), inputTypes.stream().map(Type::getTypeSignature).collect(toImmutableList())),
+                createInputParameterMetadata(type),
+                INPUT_FUNCTION.bindTo(type),
+                COMBINE_FUNCTION,
+                OUTPUT_FUNCTION,
+                ImmutableList.of(new AccumulatorStateDescriptor(
+                        NullableLongState.class,
+                        StateCompiler.generateStateSerializer(NullableLongState.class, classLoader),
+                        StateCompiler.generateStateFactory(NullableLongState.class, classLoader))),
+                VARBINARY);
+
+        Class<? extends Accumulator> accumulatorClass = AccumulatorCompiler.generateAccumulatorClass(
+                Accumulator.class,
+                metadata,
+                classLoader);
+        Class<? extends GroupedAccumulator> groupedAccumulatorClass = AccumulatorCompiler.generateAccumulatorClass(
+                GroupedAccumulator.class,
+                metadata,
+                classLoader);
+        return new BuiltInAggregationFunctionImplementation(NAME, inputTypes, ImmutableList.of(BIGINT), VARBINARY,
+                true, false, metadata, accumulatorClass, groupedAccumulatorClass);
+    }
+
+    private static List<ParameterMetadata> createInputParameterMetadata(Type type)
+    {
+        return ImmutableList.of(new ParameterMetadata(STATE), new ParameterMetadata(NULLABLE_BLOCK_INPUT_CHANNEL, type), new ParameterMetadata(BLOCK_INDEX));
+    }
+
+    public static void input(Type type, NullableLongState state, Block block, int position)
+    {
+        // do nothing
+    }
+
+    public static void combine(NullableLongState state, NullableLongState otherState)
+    {
+        // do nothing
+    }
+
+    public static void output(NullableLongState state, BlockBuilder out)
+    {
+        VARBINARY.writeSlice(out, OUTPUT);
```

Comment:
VARCHAR output type probably makes more sense.  You could make it boolean (always returns true) if you want it to do even less work.